ADR/ADR028-use-notify-for-users.md

```diff
@@ -0,0 +1,20 @@
+# ADR029: Use GOV.UK Notify for emails sent to users for simple notifications
+
+Date: 2024-03-28
+
+## Status
+
+Accepted
+
+
+## Context
+
+Emails to users that confirm they have submitted their form, and as notifications for their user account management (such as when they are upgraded from a trial to full user account, need to be sent. We currently only use Amazon SES for submission emails where TLS requirements are higher. This is an addendum to [ADR019](ADR019-user-amazon-ses.md).
```

Comment:
A closing bracket is missing. Also AFAIK we're currently only using AWS SES to send the Auth0 OTPs. [Submission emails](https://github.com/alphagov/forms-runner/blob/main/app/mailers/form_submission_mailer.rb) and [submission confirmation emails](https://github.com/alphagov/forms-runner/blob/main/app/mailers/form_submission_confirmation_mailer.rb) in runner and the [OTP to verify a change in submission email via admin](https://github.com/alphagov/forms-admin/blob/main/app/mailers/submission_email_mailer.rb) are sent via Notify aren't they?

Reply:
Sorry I got confused and made this at the end of the week before Easter hol - will amend. Thanks!

Reply:
odd because I tl;dr that Thursday afternoon 🫠

Reply:
@danworth if it's ok, I've lifted your description